packages/bot/src/scratch/hooks/toolbox.js

```diff
@@ -1,6 +1,7 @@
 import React         from 'react';
 import ReactDOM      from 'react-dom';
 import { ArrowIcon } from '../../components/Icons.jsx';
+import { flyout }    from '../../stores';
```

Comment:
@aaron-binary im not sure if we have to use react stores here but not so sure though:))

Reply:
We need to access the store outside React to make the link between Blockly and our custom Flyout component.